docs/sources/k6/v1.0.x/results-output/real-time/datadog.md

```diff
@@ -44,6 +44,8 @@ docker run --rm -d \
 
 {{< /code >}}
 
+For `DD_SITE`, be sure to use the correct datadog region url.
+
 Replace `<YOUR_DATADOG_API_KEY>` with your [Datadog API key](https://app.datadoghq.com/account/settings#api).
 
 If your account is registered with Datadog EU, change the value of `DD_SITE` to `datadoghq.eu`.
```

Comment:
```suggestion
For `DD_SITE`, make sure to use the correct [Datadog site URL](https://docs.datadoghq.com/getting_started/site/#access-the-datadog-site).

Replace `<YOUR_DATADOG_API_KEY>` with your [Datadog API key](https://app.datadoghq.com/account/settings#api).
```

I think we can refer to the Datadog docs here, and then also get rid of the specific line about Datadog EU to make things simpler.